lib/_stream_transform.js

```diff
@@ -208,18 +206,15 @@ function done(stream, er, data) {
   if (er)
     return stream.emit('error', er);
 
-  if (data !== null && data !== undefined)
+  if (data != null)
```

Comment:
would you mind adding a comment that this matches both `null` and `undefined`, and it is done on purpose?